lib/sidekiq/form526_backup_submission_process/submit.rb

```diff
@@ -24,9 +24,9 @@ class Submit
       def perform(form526_submission_id)
         return unless Settings.form526_backup.enabled
 
-        job_status = Form526JobStatus.create!(job_class: 'BackupSubmission', status: 'pending',
-                                              form526_submission_id:, job_id: jid)
         begin
```

Comment:
begin is redundant,  we can just remove it in this case and leave the rescue at the method definition level.